templates/web-application/package.json

```diff
@@ -18,58 +18,69 @@
         "reset:modules": "pnpm dlx rimraf node_modules pnpm-lock.yaml",
         "analyze": "webpack --config webpack.build.js --analyze",
         "test": "jest",
+        "test-ci": "jest --ci --coverage --reporters='jest-junit'",
```

Comment:
I don't think we want to add a code coverage report. This is not a metric that we usually value because it's not a good reflection of tests quality.